index.js

```diff
@@ -29,18 +34,26 @@ function JSHinter (inputNode, options) {
 
   for (var key in options) {
     if (options.hasOwnProperty(key)) {
-      this[key] = options[key]
+      this[key] = options[key];
     }
   }
-};
+}
 
 JSHinter.prototype.extensions = ['js'];
 JSHinter.prototype.targetExtension = 'lint.js';
 
-JSHinter.prototype.baseDir = function() {
+JSHinter.prototype.baseDir = function () {
   return __dirname;
 };
 
+JSHinter.prototype.processIgnoredFiles = function () {
+  var funnel = Remove(this.outputPath, {
+    paths: this.ignoredFiles || []
+  });
+  var builder = new broccoli.Builder(funnel);
```

Comment:
We should not be invoking the `broccoli.Builder` in the plugin itself.  Also, I do not believe that `broccoli` is imported (it is a devDep only).